dfsclient/src/main/java/org/hpccsystems/dfs/client/FileUtility.java

```diff
@@ -129,6 +161,15 @@ public boolean hasOperation()
 
         public void startOperation(String operationName)
         {
+            if (taskSpan != null)
+            {
+                taskSpan = Utils.createChildSpan(taskSpan, operationName);

```

Comment:
does it makes sense to have a single Utils.createSpan(parentSpan, spanName)? If parentSpan != null, it sets it as the parent